src/libs/PolicyUtils.js

```diff
@@ -54,9 +54,29 @@ function getPolicyBrickRoadIndicatorStatus(policy, policyMembers) {
     return '';
 }
 
+/**
+ * Check if the policy can be displayed
+ * If offline, always show the policy pending deletion.
+ * If online, show the policy pending deletion only if there is an error.
+ * @param {Object} policy
+ * @param {boolean} isOffline
+ * @returns {boolean}
+ */
+function shouldShowPolicy(policy, isOffline) {
+    return policy
+    && policy.type === CONST.POLICY.TYPE.FREE
+    && policy.role === CONST.POLICY.ROLE.ADMIN
+    && (
+        isOffline
```

Comment:
@parasharrajat I think in this way it can create race condition. Like if shouldShowPolicy function call before updation of value isNetworkOffline. because both are independent.

Reply:
I have just tested it with subscription it's not working correctly. It takes time to show welcome message.

Reply:
@parasharrajat comment added